### PR TITLE
Feature:(Backend):Test for the app

### LINKED ISSUE #6: Modularize your code
It is always advisable to create succinct functions that carry out the target logic, here converting from °F to °C, and vice versa. Thus, you can add tests for them -- and you should.

### PR DESCRIPTION
**Pull request by Ngha-Boris**
I wrote a test for the application

### CODE REVIEW

**Comment by Ngha-Boris** [comment]
Hello sir @IngridPuppet  i wrote the test as required but am having some issues please can you check.

**Comment by Ngha-Boris** [comment]
Hello sir @IngridPuppet

File: converter/tests/integration_test.rs
Comment (IngridPuppet) [inline converter/tests/integration_test.rs:1]: Your program is a binary crate, not a library crate. You can't really test it in this way. Just move the entire `tests` module at the end of the `src/conversion.rs` file. This is the idiomatic approach to unit testing. Also, do not forget to mark up the first test with `#[test]` as well.

Comment (Ngha-Boris) [reply converter/tests/integration_test.rs:1]: Ok thank you sir @IngridPuppet i will do just that.

**Comment by Ngha-Boris** [comment]
Hello sir @IngridPuppet i have finished the test and it is working please review

**Review by IngridPuppet — APPROVED** [review]
LGTM

**Pull request merged**